### PR TITLE
Fix the indention and quoting and make the choosing of the index generic

### PR DESCRIPTION
**Pull request by OrpazFishel**

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -13,16 +13,47 @@
...
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact about myself to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const facts =
+      ["I'm one of a triplet - and we are all females",
```
Comment (nolaviz) [inline portfolio/src/main/webapp/script.js:20]: Please use consistent quoting and indentation in each block of code (here the last line breaks both).

Having clean code makes it easier to read later.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -13,16 +13,47 @@
...
+  const factContainer = document.getElementById('fact-container');
+  factContainer.innerText = fact;
 }
+
+/**
+ * Generates a URL for a random image in the images directory and adds an img
+ * element with that URL to the page.
+ */
+function randomizeImage() {
+  // The images directory contains 6 images, so generate a random index between
```
Comment (nolaviz) [inline portfolio/src/main/webapp/script.js]: Why don't you just use images.length here? (It's more maintainable, e.g. if you add more images).

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -13,16 +13,47 @@
...
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const facts =
+      ["I'm one of a triplet - and we are all females",
+      "My big brothers are twins",
+      "The names of my entire family start with 'א'",
+       'I have played handball for 8 years, 13 games seasons'];
```
Comment (nolaviz) [inline portfolio/src/main/webapp/script.js]: Please use consistent quoting and indentation in each block of code (here the last line breaks both).

Having clean code makes it easier to read later.

**Review by nolaviz — APPROVED** [review]

**Review by nolaviz — APPROVED** [review]

**Comment by OrpazFishel** [comment]
Create a personal web app

**Pull request merged**